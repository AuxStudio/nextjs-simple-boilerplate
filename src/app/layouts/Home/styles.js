import css from 'styled-jsx/css'; // eslint-disable-line

const styles = css`
  .container {
    height: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
  }
`;

export default styles;
