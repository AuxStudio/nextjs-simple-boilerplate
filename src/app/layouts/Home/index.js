import React from 'react';
import PropTypes from 'prop-types';

import styles from './styles';

import Layout from '../../components/Layout';
import Typography from '../../components/Typography';

const Home = () => {
  return (
    <Layout>
      <div className="container">
        <Typography type="title">Hello World</Typography>
      </div>

      <style jsx>{styles}</style>
    </Layout>
  );
};

Home.propTypes = {};
Home.defaultProps = {};

export default Home;
