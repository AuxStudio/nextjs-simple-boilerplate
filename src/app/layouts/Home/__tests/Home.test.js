import React from 'react';
import renderer from 'react-test-renderer';

import Home from '..';

describe('Home', () => {
  describe('renders', () => {
    it('renders with minimum required props', () => {
      const component = renderer.create(<Home />);

      expect(component).toMatchSnapshot();
    });
  });
});
