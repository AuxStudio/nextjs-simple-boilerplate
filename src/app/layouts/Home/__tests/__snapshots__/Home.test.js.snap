// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders renders with minimum required props 1`] = `
<div
  className="jsx-1031917824 container flex flex-center"
>
  <h1
    className="jsx-2736794252"
  >
    Hello World
  </h1>
</div>
`;
